proto/evmos/fees/v1/query.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+package evmos.fees.v1;
+
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "evmos/fees/v1/genesis.proto";
+import "evmos/fees/v1/fees.proto";
+import "google/api/annotations.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/tharsis/evmos/v3/x/fees/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // FeesContracts retrieves registered fees
+  rpc FeesContracts(QueryFeesContractsRequest) returns (QueryFeesContractsResponse) {
+    option (google.api.http).get = "/evmos/fees/v1/fees";
+  }
+
+  // FeesContract retrieves a registered fee
+  rpc FeesContract(QueryFeesContractRequest) returns (QueryFeesContractResponse) {
+    option (google.api.http).get = "/evmos/fees/v1/fees/{contract}";
+  }
```

Comment:
we should consider incorporating a query that returns all the registered contracts per withdraw address/deployer.